javascript/javascript1/week1/index.js

```diff
@@ -13,23 +13,32 @@ console.log('I will be ' + age + ' years old in ' + yearFuture);
 
 // Goodboy-Oldboy (A dog age calculator)
 
-const dogYearOfBirth = prompt('What is your dogs year of birth?');
+const dogYearOfBirth = +prompt('What is your dogs year of birth?');
 const dogYearFuture = 2027;
 const dogYears = dogYearFuture - dogYearOfBirth;
 const shouldShowResultInDogYears = prompt(
   'Would you like the result to be shown in dog years? Type either "Yes" or "No"'
 ).toLowerCase();
 
-if (shouldShowResultInDogYears === 'yes') {
-  console.log(
-    'Your dog will be ' + dogYears * 7 + ' dog years old in ' + dogYearFuture
-  );
-} else if (shouldShowResultInDogYears === 'no') {
-  console.log(
-    'Your dog will be ' + dogYears + ' human years old in ' + dogYearFuture
-  );
+if (
+  isNaN(dogYearOfBirth) ||
+  dogYearOfBirth > 2023 ||
+  dogYearOfBirth < 1990 ||
+  dogYearOfBirth == null
```

Comment:
Good. For the future, if you check value with isNaN method, you do not need to check it for null, because isNaN does it for you